anndata/_core/sparse_dataset.py

```diff
@@ -376,9 +376,9 @@ def to_backed(self) -> BackedSparseMatrix:
     def to_memory(self) -> ss.spmatrix:
         format_class = get_memory_class(self.format_str)
         mtx = format_class(self.shape, dtype=self.dtype)
-        mtx.data = self.group["data"][...]
-        mtx.indices = self.group["indices"][...]
-        mtx.indptr = self.group["indptr"][...]
+        mtx.data = self.group["data"]
+        mtx.indices = self.group["indices"]
+        mtx.indptr = self.group["indptr"]
```

Comment:
Is this neccesary? Probably leading to test failures. Also basically what `to_backed` is for?